actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -40,11 +40,14 @@ def url_for(options = nil) # :nodoc:
         end
       end
 
+      # Returns previous URL in the browser history from request referrer or Javascript history back method
       def _back_url # :nodoc:
         _filtered_referrer || 'javascript:history.back()'
       end
       protected :_back_url
 
+      # Returns referrer URL of the current request and also ensures it doesn't contain any Javascript protocol
+      # Raises an exception +URI::InvalidURIError+ when referrer has invalid URI format
```

Comment:
Are you sure an error is raised, it looks like it is `rescue`d in the lines below :arrow_down: